app/src/navigation/screens/ClaimScreen.tsx

```diff
@@ -152,10 +151,20 @@ const ClaimScreen: React.FC = () => {
   };
   const openVerifyOtpScreen = async () => {
     setLoading(true);
-    const { countryCode, number } = formatMobileNumberState();
+
+    const newMobileState = formatMobileNumberState();
+    if (newMobileState.countryCode.trim() !== "+61") {
+      Alert.alert(
+        "Error",
+        "IDEM only supports Australian numbers for mobile claims/verification"
+      );
+      setLoading(false);
+      return;
+    }
+
     try {
       const otpResponse = await api.requestOtp({
-        mobileNumber: `${countryCode}${number}`
+        mobileNumber: `${newMobileState.countryCode}${newMobileState.number}`
```

Comment:
countryCode here still isn't getting trim

maybe we should trim in formatMobileNumberState function instead
so both line 156 and line 167 country code is trimmed